crates/matrix-sdk-sqlite/src/state_store.rs

```diff
@@ -2442,6 +2465,29 @@ mod migration_tests {
         txn.prepare_cached("INSERT INTO send_queue_events (room_id, room_id_val, transaction_id, content, wedged) VALUES (?, ?, ?, ?, ?)")?
             .execute((room_id_key, room_id_value, transaction_id.to_string(), content, is_wedged))?;
 
+        Ok(())
+    }
+    fn add_dependent_send_queue_event_v7(
```

Comment:
I wish we had automatic rustfmt rules for this, but: we try to keep blank lines between functions, to visually separate them (and make moving from paragraph to paragraph fast, in editors like vim :nerd_face:)

```suggestion
    }
    
    fn add_dependent_send_queue_event_v7(
```